scrapers/reddit.js

```diff
@@ -0,0 +1,56 @@
+var BaseLoklakScrapper = require('./base');
+
+class RedditLoklakScrapper extends BaseLoklakScrapper {
+
+    constructor() {
+        super('Reddit', 'https://www.reddit.com');
+    }
+
+    argumentSanityCheck(args) {
+        super.argumentSanityCheck(args);
+
+        if (args.length <= 2) {
+            console.error('Atleast one argument required.');
+            process.exit(-1);
+        }
+
+        return true;
+    }
+
+    onInit() {
+        var profile = process.argv[2];
```

Comment:
Don't access the process directly, use `PROC_ARGS` or `SLICED_PROC_ARGS` as we might change the Base scrapper to be independent of processes and more like an object which can run as a node script as well as can be included in the other places as objects.